ios/AzureNotificationHubIOS.js

```diff
@@ -0,0 +1,445 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule AzureNotificationHubIOS
+ * @flow
+ */
+'use strict';
+
+import {NativeEventEmitter} from 'react-native';
+import {NativeModules} from 'react-native';
+
+const RCTAzureNotificationHubManager = NativeModules.AzureNotificationHubManager;
+const invariant = require('../../react-native/node_modules/fbjs/lib/invariant');
+
+const PushNotificationEmitter = new NativeEventEmitter(RCTAzureNotificationHubManager);
+
+const _notifHandlers = new Map();
+
+const DEVICE_NOTIF_EVENT = 'remoteNotificationReceived';
```

Comment:
I don't think we need since they're unique events that can only be received in the AppDelegate class. That's why we need to modify the [AppDelegate.m](https://facebook.github.io/react-native/docs/pushnotificationios.html) file for every react-native app supporting push notifications.